src/app/models/indicator.model.ts

```diff
@@ -10,6 +10,7 @@ export class Indicator {
     available_units?: string[];
     default_units?: string;
     parameters?: IndicatorParameter[];
+    thresholdIndicator: boolean;
```

Comment:
Was thinking myself of moving this as well as some other parameter-specific configuration to a shared constants file for all the extra parameters, mentioned in the notes.